### DISCUSSION TITLE
summarize

### DISCUSSION THREAD

**Discussion by GustavoSantosRodrigues** [filamentphp/filament#13112] [Help] [answered]
How can I change the name that says "SUMMARY" or delete
THIS WOULD BE MY LAST LINE, I WOULD LIKE TO CHANGE THE SUMMARY AND ADD ANOTHER ONE OR TAKE IT OUT
![image](https://github.com/filamentphp/filament/assets/93735442/54b02f89-c6a4-44f7-8fa2-c1bb9e726779)

**Comment by leandrocfe** [comment, marked as answer]
publish the table translations - `php artisan vendor:publish --tag=filament-tables-translations`

edit the summary->heading
```php
'summary' => [

    'heading' => 'Resumo',
```

**Reply by farhadrahmanii** [reply]
Thank you so much this was very helpful sir
